packages/gitbook/src/lib/visitors.ts

```diff
@@ -30,6 +31,25 @@ type VisitorAuthCookieValue = {
     token: string;
 };
 
+type ClaimPrimitive =
+    | string
+    | number
+    | boolean
+    | null
+    | undefined
+    | { [key: string]: ClaimPrimitive }
+    | ClaimPrimitive[];
+
+/**
+ * The result of a visitor info lookup that can include:
+ *   - a visitor token (JWT)
+ *   - a record of visitor public/unsigned claims (JSON object)
+ */
+export type VisitorPayloadLookup = {
```

Comment:
Instead of creating another new type, can we use `SiteVisitorPayload`?

Reply:
Not really because the lookup here also returns info about where the token was retrieved from (params, cookies etc)